recipes/python-ftfy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "python-ftfy" %}
+{% set pypi_name = "ftfy" %}
+{% set version = "4.3.1" %}
+{% set sha256sum = "60af7f4566a966fcb9681f3c93814017e981f5bbfffc2f5f33aadf8238101f4b" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - html5lib
+    - wcwidth
```

Comment:
Are all of these needed at build time? Guessing we only need `python` and `setuptools`.